src/Tasks/GenerateLauncher.cs

```diff
@@ -50,12 +52,22 @@ public override bool Execute()
 
             if (EntryPoint == null)
             {
-                Log.LogErrorWithCodeFromResources("GenerateLauncher.EmptyEntryPoint");
+                Log.LogErrorWithCodeFromResources("GenerateLauncher.InvalidInput");
```

Comment:
Why this removal?

Reply:
I had added EmptyEntryPoint in a recent commit. It was pointed to me by Nikola that we already have a string for the same error message.